### PR TITLE
AT DSL - Removing WaitCondition, consistently applying Condition instead

### PR DESCRIPTION
**Pull request by CjHare**
## PR description
WaitCondition is really a Condition :. removing WaitCondition

### CODE REVIEW

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java [hunk 1]
```diff
@@ -37,21 +37,21 @@ public void setUp() throws Exception {
   @Test
   public void shouldBeAbleToGetValidatorsForBlockNumber() {
     cluster.verify(clique.validatorsAtBlockEqual("0x0", minerNode1));
-    minerNode1.waitUntil(wait.chainHeadIsAtLeast(1));
+    minerNode1.verify(blockchain.minimumHeight(1));
 
     minerNode1.execute(cliqueTransactions.createAddProposal(minerNode2));
-    cluster.waitUntil(wait.chainHeadHasProgressedByAtLeast(minerNode1, 1));
+    cluster.verify(blockchain.minimumHeightProgression(minerNode1, 1));
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java]: progression sounds a bit odd here. Is there a consistent phrasing or tense used or meant to be used for the dsl?

Comment (CjHare) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java]: suggested alternative?

Comment (CjHare) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java]: `blockchain.reachesHeight()`

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueProposeRpcAcceptanceTest.java [hunk 2]
```diff
@@ -35,10 +35,10 @@ public void shouldAddValidators() throws IOException {
     cluster.start(minerNode1, minerNode2, minerNode3);
 
     cluster.verify(clique.validatorsEqual(minerNode1, minerNode2));
-    final WaitCondition cliqueValidatorsChanged = wait.cliqueValidatorsChanged(minerNode1);
+    final Condition cliqueValidatorsChanged = clique.awaitSignerSetChange(minerNode1);
     minerNode1.execute(cliqueTransactions.createAddProposal(minerNode3));
     minerNode2.execute(cliqueTransactions.createAddProposal(minerNode3));
-    cluster.waitUntil(cliqueValidatorsChanged);
+    cluster.verify(cliqueValidatorsChanged);
```
Comment (rain-on) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueProposeRpcAcceptanceTest.java]: This line should be removed - if we're ditching the "waiting" concept, then no need to ensure miner1 has changed - just ensure all the cluster have in the next line (and thus line 38 can probably go too).

Comment (CjHare) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueProposeRpcAcceptanceTest.java]: done

**Pull request merged**